apis/login.php

```diff
@@ -0,0 +1,56 @@
+<?php
+session_start();
+include 'db.php';
+// echo $email;
+if($_SERVER["REQUEST_METHOD"] === "POST" ){
+
+    $loginEmail=$_POST['loginEmail'];
+    $loginPassword=$_POST['loginPassword'];
+
+  	  if(!empty($loginEmail) && !empty($loginPassword))
+   	 {
+          $loginPassword=hash('sha512', $loginPassword);
+
+       		 $query1 ="SELECT * FROM admin WHERE email='$loginEmail' AND password='$loginPassword'";
+       		 $query1_run=mysqli_query($con,$query1);
+		        if(mysqli_num_rows($query1_run)==1)
+       		 { 
+                  $query_row = mysqli_fetch_array($query1_run,MYSQLI_ASSOC);
+                  $_SESSION['logged_in'] = 1;
+                  $_SESSION['name'] = $query_row['name'];
+                  $_SESSION['admin_id']=$query_row['admin_id'];
+                  $_SESSION['Email']=$query_row['email'];
+                  $admin_id = $_SESSION['admin_id'];
+                  $basicInfo=[];
+                  
+                 
+                 $query = mysqli_query($con, "SELECT * FROM admin where admin_id ='".$admin_id."'");
```

Comment:
This is not required. We can also get the details from the $query_row and store it in basic info